File: mycollab-mobile/src/main/java/com/esofthead/mycollab/mobile/module/crm/events/CrmEvent.java
package com.esofthead.mycollab.mobile.module.crm.events;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;

/**
 * @author MyCollab Ltd.
 * @since 3.0
 */
public class CrmEvent {

    public static class GotoHome extends ApplicationEvent {
        public GotoHome(Object source, Object data) {
            super(source, data);
        }
    }
}

File: mycollab-mobile/src/main/scala/com/mycollab/mobile/module/crm/events/CrmEvent.scala
package com.mycollab.mobile.module.crm.events

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd
  * @since 5.4.9
  */
object CrmEvent {

  class GotoActivitiesView(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}
